src/main/java/org/jenkinsci/plugins/saml/SamlSecurityRealm.java

```diff
@@ -289,6 +289,29 @@ private Saml2Client newClient() {
     return client;
   }
 
+  /**
+   * @see SecurityRealm#getPostLogOutUrl
+   * Note: It does not call the logout service on SAML because the library does not implement it on this version,
+   * it will be done when we upgrade the library.
+   */
+  @Override
+  protected String getPostLogOutUrl(StaplerRequest req, Authentication auth) {
+    LOG.log(Level.FINE,"Doing Logout");
```

Comment:
probably want the curent authentication name (username) in here so this message is actually useful :)